### PR TITLE
Story/vspc 157

### PR DESCRIPTION
**Pull request by pradeep-017**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
Created a button "Edit Image" on display page of image and then used EditImageController and Edit page of image to edit name/description of images

...Put description here...
  
## Anything else the reviewer needs to know?

... describe here ...

### CODE REVIEW

File: vspace/src/main/webapp/WEB-INF/views/staff/images/edit.html [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
-    <div layout:fragment="content">
+    <div class="main-content" layout:fragment="content">
         <h1>Edit Image: [[${imageForm.fileName}]]</h1>
-        <form th:action="@{'/staff/image/'+${imageId}+'/edit'}"
-            method="POST" name="f">
-            <input type="hidden" name="${_csrf.parameterName}"
-                value="${_csrf.token}" />
+        
+        <form method="POST"
+        	th:action='@{|/staff/image/${imageId}/edit?${_csrf.parameterName}=${_csrf.token}|}' th:object="${imageForm}"
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/images/edit.html:7]: why did you move the csrf token here? it should stay in the body.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Resolve conflicts please

File: vspace/src/main/webapp/WEB-INF/views/staff/images/edit.html (hunk 1 above)
Comment (pradeep-017) [reply vspace/src/main/webapp/WEB-INF/views/staff/images/edit.html]: As discussed over zoom, I have removed this and now the form is working.

File: vspace/src/main/webapp/WEB-INF/views/staff/images/image.html [hunk 2]
```diff
@@ -22,7 +22,11 @@ <h1>
         <p style="max-height: 100px; overflow-y: scroll;">
             <b>Description:</b> [[${image.description}]]
         </p>
-        <div id="image">
+        
+		<a th:href="@{'/staff/image/'+${image.id}+'/edit'}"
+                class="btn primary-btn">Edit Image</a>
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/images/image.html:27]: indentation

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
After that should be good.

File: vspace/src/main/webapp/WEB-INF/views/staff/images/image.html (hunk 2 above)
Comment (pradeep-017) [reply vspace/src/main/webapp/WEB-INF/views/staff/images/image.html:27]: fixed!
